Structure_Engine/Query/Mass.cs

```diff
@@ -30,13 +33,19 @@ public static partial class Query
         /**** Public Methods                            ****/
         /***************************************************/
 
+        [Description("Calculates the mass of a Bar as its length times mass per metre. The mass per metre is generally takes as section area times density. No offsets or similar ar taken into acount.")]
```

Comment:
Calculates the mass of a Bar as its length times mass per metre. The mass per metre is calculated as section area multiplied by the density. No offsets or similar ar taken into acount.